recompletion.php

```diff
@@ -102,10 +116,14 @@
         } else {
             if ($name == 'recompletionemailsubject' || $name == 'recompletionemailbody') {
                 $value = '';
+            } else if ($name === 'nextresettime') {
```

Comment:
as long as you are refactoring some part of the code, maybe you can rewrite this multiple if else block using switch?

Reply:
Thanks Dima, I'll see if there's a better way to clean up this section. Appreciate the feedback.